cpp/src/strings/strings_column_factories.cu

```diff
@@ -18,92 +18,23 @@
 #include <cudf/column/column_factories.hpp>
 #include <cudf/detail/nvtx/ranges.hpp>
```

Comment:
You removed all of the `CUDF_FUNC_RANGE();` calls to the `detail::make_strings_column`. Usually these are put in the non-detail function right before calling the detail version.